conans/client/rest/rest_client_v2.py

```diff
@@ -243,3 +245,21 @@ def remove_packages(self, ref, package_ids=None):
             url = self.conans_router.remove_package(pref)
             self.requester.delete(url, auth=self.auth, headers=self.custom_headers,
                                   verify=self.verify_ssl)
+
+    def get_recipe_revisions(self, ref):
+        url = self.conans_router.recipe_revisions(ref)
+        data = self.get_json(url)
+        return self._format_dates_in_revision_list(data)
+
+    def get_package_revisions(self, pref):
+        url = self.conans_router.package_revisions(pref)
+        data = self.get_json(url)
+        return self._format_dates_in_revision_list(data)
+
+    @staticmethod
+    def _format_dates_in_revision_list(data):
+        ret = {"reference": data["reference"],
+               "revisions": [{"revision": r["revision"],
+                              "time": from_iso8601_to_datetime(r["time"])}
```

Comment:
Seems weird to do a conversion from the datetime of the server. Shouldn't be that datetime like constant everywhere, just format it for output?

Reply:
I'm not sure I understand. The date from the server is in `ISO-8601` format (string), it is a strong recommendation when sharing dates. What are you suggesting exactly?

Reply:
I am saying that I don't understand why the rest_client should be doing this data conversion. If that time was going to be written to disk, then it should be converted back to the format given by the server? Probably I am missing something, just want to understand the flow.